datasets/russian_super_glue/russian_super_glue.py

```diff
@@ -0,0 +1,527 @@
+          url: `string`, url for information about the data set.
+          label_classes: `list[string]`, the list of classes for the label if the
+            label is present as a string. Non-string labels will be cast to either
+            'False' or 'True'.
+          **kwargs: keyword arguments forwarded to super.
+        """
+        # 0.0.1: Initial version.
+        super(RussianSuperGlueConfig, self).__init__(version=self.VERSION, **kwargs)
+        self.features = features
+        self.label_classes = label_classes
+        self.data_url = data_url
+        self.citation = citation
+        self.url = url
+
+
+class RussianSuperGlue(datasets.GeneratorBasedBuilder):
+
+    BUILDER_CONFIGS = [
+        RussianSuperGlueConfig(
+            name="lidirus",
+            description=_LIDIRUS_DESCRIPTION,
+            features=[
+                "sentence1",
+                "sentence2",
+                "knowledge",
+                "lexical-semantics",
+                "logic",
+                "predicate-argument-structure",
+            ],
+            label_classes=["entailment", "not_entailment"],
+            data_url="https://russiansuperglue.com/tasks/download/LiDiRus",
+            citation="",
+            url="https://russiansuperglue.com/tasks/task_info/LiDiRus",
+        ),
+        RussianSuperGlueConfig(
+            name="rcb",
+            description=_RCB_DESCRIPTION,
+            features=["premise", "hypothesis", "verb", "negation"],
+            label_classes=["entailment", "contradiction", "neutral"],
+            data_url="https://russiansuperglue.com/tasks/download/RCB",
+            citation="",
+            url="https://russiansuperglue.com/tasks/task_info/RCB",
+        ),
+        RussianSuperGlueConfig(
+            name="parus",
+            description=_PARUS_DESCRIPTION,
+            label_classes=["choice1", "choice2"],
+            features=["premise", "choice1", "choice2", "question"],
+            data_url="https://russiansuperglue.com/tasks/download/PARus",
+            citation="",
+            url="https://russiansuperglue.com/tasks/task_info/PARus",
+        ),
+        RussianSuperGlueConfig(
+            name="muserc",
+            description=_MUSERC_DESCRIPTION,
+            features=["paragraph", "question", "answer"],
+            data_url="https://russiansuperglue.com/tasks/download/MuSeRC",
+            citation=_MUSERC_CITATION,
+            url="https://russiansuperglue.com/tasks/task_info/MuSeRC",
+        ),
+        RussianSuperGlueConfig(
+            name="terra",
+            description=_TERRA_DESCRIPTION,
+            features=["premise", "hypothesis"],
+            label_classes=["entailment", "not_entailment"],
+            data_url="https://russiansuperglue.com/tasks/download/TERRa",
+            citation="",
+            url="https://russiansuperglue.com/tasks/task_info/TERRa",
+        ),
+        RussianSuperGlueConfig(
```

Comment:
This configuration should have `label_classes ` no ? I can see the `label` field in the examples in the readme

Reply:
Do you mean the TERRa dataset? The label classes are there.

Reply:
What I mean is that we could name the label classes of RUSSE, RWSD and DANETQA: "same sense"/"not same sense", "cofererence"/"not coreference", "yes"/"no" via the `label_classes` parameter that you can pass to the builder config.

It's useful to add this info to know what the label represents programmatically, since the label names are used to define the ClassLabel feature type of the `label` column of your data.

Reply:
Are the label names supposed to be used for inference? Conceptually, I mean. Submissions on the leaderboard are expected to be in particular format (such as `idx: 1, label: "true"`), so I was thinking that it would be useful to be able reuse label names for that. Here, and in the original SuperGLUE, the datasets with binary answers are generally labeled with boolean values. However, now I noticed that these values are used inconsistently in the sample submission. It appears as "true"/"false", "True"/"False", and 0/1.

Anyway, my question is are label names meant to be just supplementary info?

Meanwhile, I'll have to check that the code written so far works as intended with inconsistent label names.

Reply:
So the choice is to use whatever label names are expected in the submission or make the names meaningful. I'm not so sure that it's an obvious one in this case.

Reply:
I didn't know about the expected names in the submission. We should probably keep the names from what's expected in the submission then.

Reply:
In doubt maybe we can just stay consistent with the `super_glue` dataset script, i.e. don't have `label_classes` for those tasks. Does that sound good to you ?

If yes, I guess we're done since it doesn't have `label_classes` for those tasks already.